packages/devtools_app/lib/src/framework/release_notes/release_notes.dart

```diff
@@ -76,66 +75,121 @@ class ReleaseNotesController extends SidePanelController {
   Future<void> _fetchAndShowReleaseNotes({
     SemanticVersion? versionFloor,
   }) async {
+    if (_debugReleaseNotesUrl case final debugUrl?) {
+      // Specially handle the case where a debug release notes URL is specified.
+      final debugUri = Uri.parse(debugUrl);
+      final releaseNotesMarkdown = await http.read(
+        debugUri,
```

Comment:
style nit: you can remove trailing comma from dart code as long as this fits on one line